content/systems/filecoin_vm/sysactors/reward_actor.md

```diff
@@ -1,18 +1,20 @@
 ---
 title: RewardActor
+weight: 4
 dashboardWeight: 2
-dashboardState: incomplete
+dashboardState: stable
 dashboardAudit: 0
 dashboardTests: 0
 ---
 
 # RewardActor
 ---
 
-RewardActor is where unminted Filecoin tokens are kept. RewardActor contains a `RewardMap` which is a mapping from owner addresses to `Reward` structs. 
+The `RewardActor` is where unminted Filecoin tokens are kept. The `RewardActor` contains a `RewardMap` which is a mapping from owner addresses to `Reward` structs. 
```

Comment:
This `RewardMap` stuff is out of date. The actor distributes rewards directly to miner actors, where they are locked for vesting.

The paragraph below is wrong too.